File: helpdeskangular/src/app/payload/CommentResponse.ts
// 
export class CommentResponse {

    constructor(

        public commentid: number,

        // comment content
        public commentDescription: string,

        public commenterid: number,

        public commenterName: string,

        public commenterPhone: string,

        public commenterEmail: string,

        // commenter = commenterid + commenterName + commenterPhone + commenterEmail
        public commenter: string,

        public commentDatetime: string,
        public customFilename: string,
        public originalFilename: string

    ) {

    }

}

File: helpdeskspringboot/src/main/java/com/ez/payload/CommentResponse.java
package com.ez.payload;

import java.util.Date;

public interface CommentResponse {

    // "commentid" column
    Long getCommentid();

    // "commentDescription" column
    String getCommentDescription();

    // "commenterid" column
    Long getCommenterid();

    // "commenterName" column
    String getCommenterName();

    // "commenterPhone" column
    String getCommenterPhone();

    // "commenterEmail" column
    String getCommenterEmail();

    // "commenter" column.
    // commenter = commenterid + commenterName + commenterPhone + commenterEmail
    String getCommenter();

    // "commentDatetime" column
    Date getCommentDatetime();

    // "getCommentCustomFilename" column
    String getCommentCustomFilename();

    // "originalFilename" column
    String getOriginalFilename();

}
